packages/design/assets/index.css

```
@@ -0,0 +1,253 @@
+
+    --vc-color-on-warning: var(--vc-color-neutral-950);
+}
+
+/* -----------------------------------------------------------------
+   Tailwind v4 @theme block — expose vuecs tokens as Tailwind
+   color/radius names so components can write e.g. `bg-primary-600`,
+   `text-fg`, `rounded-md` and have them resolve to the vc token
+   chain. Non-inline so browser-time swaps (.dark, setPalette) work.
+   ----------------------------------------------------------------- */
+
+@theme {
```

Stylelint (17.9.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)